tier1/{{cookiecutter.project_slug}}/repolinter.json

```diff
@@ -322,11 +322,11 @@
         "rule": {
           "type": "file-contents",
           "options": {
-            "globsAll": [
-              "README.md"
-            ],
+            "globsAll": ["README.md"],
             "content": "Community",
-            "flags": "i"
+            "flags": "i",
+            "file-name": "README.md",
+            "file-content": "## Community\n The {name_of_project_here} team is taking a community-first and open source approach to the product development of this tool. We believe government software should be made in the open and be built and licensed such that anyone can download the code, run it themselves without paying money to third parties or using proprietary software, and use it as they will.\n\nWe know that we can learn from a wide variety of communities, including those who will use or will be impacted by the tool, who are experts in technology, or who have experience with similar technologies deployed in other spaces. We are dedicated to creating forums for continuous conversation and feedback to help shape the design and development of the tool.\n\nWe also recognize capacity building as a key part of involving a diverse open source community. We are doing our best to use accessible language, provide technical and process documents, and offer support to community members with a wide variety of backgrounds and skillsets. \n"
```

Comment:
Change title to `Community Guidelines`

https://github.com/DSACMS/repo-scaffolder/blob/main/tier3/%7B%7Bcookiecutter.project_slug%7D%7D/README.md?plain=1#L88